docs/modules/ROOT/pages/how-to-guides/proc_managed_services_onboarding.adoc

```diff
@@ -111,13 +82,11 @@ releaseStrategy: <release-strategy> <.>
 * Run the following command to copy the application from the development workspace to the managed workspace:
 
 +
-[source,]
+[source,shell]
 ----
 ./copy-application.sh <managed workspace name> -a <development workspace name>/<application name>
 ----
 
 +
-=======
 NOTE: Run `./{ProductName}copy.sh --help` to review all the help options of the script.
```

Comment:
This renders as "Run ./App Studiocopy.sh --help to review all the help options of the script." - what script is this? Shouldn't this be the same script as above? "./copy-application.sh --help"?